merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/services/ConstraintsDSLCompilationServiceTests.java

```diff
@@ -150,7 +150,7 @@ export default function myConstraint() {
              return Real.Resource("state of charge");
           }
         """,
-        "TypeError: TS2322 Incorrect return type. Expected: 'Constraint', Actual: 'Real'."
+        "TypeError: TS2322 Incorrect return type. Expected: 'Constraint | Promise<Constraint>', Actual: 'Real'."
```

Comment:
Can you described a case where the return would be a `Promise<Constraint>`, I am getting it.

Reply:
TBH I'm not sure when our users would ever do that. The only example I can think of is if they queried some data from another server and needed to await it.

This change is just because we should be keeping up to date with the current aerie-ts-user-code-runner version, and not because it is needed for this PR. I was just messing with `package.json` and `package-lock.json` so much that I decided to update to 0.5.0.

Reply:
Yeah, this is because with V0.4.0 of the user code runner, we enabled async returns. This message is simply reflecting that that is a valid return type. You can customize the error messaging on this with the message mapper passed to the constructor if you really want to remove the Promise<> part of the message https://github.com/NASA-AMMOS/aerie-ts-user-code-runner/blob/develop/src/UserCodeRunner.ts#L24